portfolio/src/main/webapp/ChartScript.js

```diff
@@ -0,0 +1,24 @@
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(dessertChart);
+
+/** Creates a chart and adds it to the page. */
+function dessertChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Dessert');
+  data.addColumn('number', 'Count');
+        data.addRows([
+          ['Cake', 15],
+          ['Pie', 5],
+          ['Cookies', 30]
+        ]);
```

Comment:
Why all this space?

Reply:
With very little appended entries in data, it is better to put this on one line instead of line per entry